### PR TITLE
Avatar Recorder and Playback Fixes

### PR DESCRIPTION
**Pull request by thoys**
- Fixes the clip length display.
- Disables the avatars physics during a playback.
- Stops the clip after the clip reaches the end rather than pausing it. (Makes it easier to hit play again)
- Fixed the timer display, which was hidden because it was too small.
- Made the timer and slider to follow the toolbar.

## QA Test

- Test if the `developer/utilities/record/recorder.js` script works as expected.
- Test using this QA build.
- Try recording with walking and without walking. Also saving and loading recordings should be still functional.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: b21bd3afa9d22cd87ce2193493ddee3c65789e51 available at: https://gist.github.com/2a426bfd1bf3d89d755655557179bdae<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9825-b21bd3a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9825-b21bd3a.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 097db9a7f98bbab309cf30033e26c4b2d461df17 available at: https://gist.github.com/2a426bfd1bf3d89d755655557179bdae<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9825-097db9a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9825-097db9a.dmg

**Comment by claudioshg** [comment]
Testing

**Comment by claudioshg** [comment]
### Desktop PC - Windows 10 Home - CPU 4.0GHz Intel i7 - GTX1070 - RAM 16GB - Tested with Oculus Rift

Tested: Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9825-097db9a.exe

**tl;dr:** In VR and in Desktop Mode, the recorder works. I was able to use the Record, Pause/Play, Loop, Save, and Upload buttons as intended. However, in VR there are some bugs. 

While using VR:
- Any interaction that the lasers have with the recorder's UI causes the laser animation to stop and is replaced with a white mouse cursor that becomes immovable for about 3 seconds. I was still able to use all the buttons, and they executed their functions.
hifi-snap-by-claudioshg-on-2017-03-08_17-48-53
- Using the mouse on the UI works perfectly. I'm able to drag the UI anywhere and click or trigger any of the functions without any problems.
- There were many occasions where the mouse cursor would split into two and that made navigating the UI extremely hard. It was like double vision and I had no idea what I was clicking on.
- The recorder UI would become inaccessible too often. The UI didn't disappear, it was in the same spot as always, but it became unregistrable. No matter how many times the mouse went over it, the UI would not highlight and become active.

In Desktop Mode:
- Everything works great.

Notes:
- Recording while walking, jumping, and flying works great.
- I'm sure this was intended, but just covering all of my bases. Uploading a saved file does reproduce the same animation, audio, but the avatar does not re-appear in the same spot the recording took place.
